packages/stream_video/lib/src/call/call.dart

```diff
@@ -58,6 +58,7 @@ const _tag = 'SV:Call';
 
 const _reconnectTimeout = Duration(seconds: 30);
 const _fastReconnectTimeout = Duration(seconds: kDebugMode ? 10 : 3);
+const _reactionAutoDismissTime = Duration(seconds: 5);
```

Comment:
Should we somehow make this customisable?

Reply:
Added 👍